server-api/routes/products.js

```diff
@@ -0,0 +1,99 @@
+const express = require('express');
+const product = require('../models/product');
+const router = express.Router();
+
+//Getting All
```

Comment:
comments should have a space after `//`
```suggestion
// Get all products
```